frontend-v2/src/features/main/Sidebar.tsx

```diff
@@ -214,7 +218,34 @@ export default function Sidebar() {
     (step) => step !== projectsPage && step !== helpPage,
   );
 
-  const drawer = (
+  const [isExpanded, setIsExpanded] = useState(true);
+  const onCollapse = () => {
+    const element = document.getElementById("main-content");
+    element?.classList.remove("on-expand");
+    element?.classList.add("on-collapse");
+
+    const simulationsNav = document.getElementById("simulations-content");
+    if (simulationsNav) {
+      simulationsNav?.classList.remove("on-expand");
+      simulationsNav?.classList.add("on-collapse");
+    }
```

Comment:
This logic could maybe move into a function outside of the component, to make it clear that the DOM is being modified outside of React.

Also, the DOM changes here won't be tracked by React's virtual DOM, so I'm not 100% sure if they'll persist across re-renders.